### PR TITLE
3.6 support

### PR DESCRIPTION
**Pull request by keyan3**
For some reason, in the GH workflow environment, when scs is downloaded it runs its `setup.py` immediately instead of waiting until the end of the install process. The setup script installs numpy, which scs requires, but it fetches the newest version (1.20) which is not compatible with python 3.6 even if we specify <1.20 in our requirements. As far as I can tell, this only happens on GitHub (tried on Colab, local conda python, and local homebrew python without issues). 
image

To get around this we can install numpy and scipy before all other dependencies in the GitHub build. Technically, this could mask issues that arise from dependencies having numpy and scipy as build requirements like pyfim did, but I think it's unlikely we'll be using packages that have unhandled build requirements.

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        pip install 'numpy<1.20' 'scipy<1.6'
```
Comment (csinva) [inline .github/workflows/python-package.yml:25]: Is there any way to only run this line if we are using py3.6?

**Comment by csinva** [comment]
Very weird...looking at the scs package they only require numpy >= 1.7 in the first place, so no idea why this is happening.

This workaround seems fine though, especially if you can get it to only do this pip install for py3.6.

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (keyan3) [reply .github/workflows/python-package.yml]: Changed!

**Comment by csinva** [comment]
perfect!

**Pull request merged**